### PR TITLE
[INFRA/IIGO] Some Logging and Minor Bug Fixes

### PR DESCRIPTION
**Pull request by preetl**
# Summary

Several small changes related to IIGO bugs :

- logging added to rule votes, elections and monitoring
- a monitoring fix (IDs weren't initialised to IIGO role IDs)
- dead islands can no longer hold office

## Additional Information

N/A

## Test Plan

Still pass and new tests for new function.

### CODE REVIEW

File: internal/server/iigointernal/orchestration.go [hunk 1]
```diff
@@ -26,8 +26,11 @@ var monitoring monitor
 
 // RunIIGO runs all iigo function in sequence
 func RunIIGO(logger shared.Logger, g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.Client, gameConf *config.Config) (IIGOSuccessful bool, StatusDescription string) {
-	// featureJudge is an instantiation of the Judge interface
-	// with both the Base Judge features and a reference to client judges
+
+	removeDeadBodiesFromOffice(g)
```
Comment (Neelesh99) [inline internal/server/iigointernal/orchestration.go:30]: Best line of code ever !! ;)

File: internal/server/iigointernal/utilities.go [hunk 2]
```diff
@@ -66,3 +68,22 @@ func boolToFloat(input bool) float64 {
 	}
 	return 0
 }
+
+// if an IIGO role is dead, it is replaced with a random living island
+func removeDeadBodiesFromOffice(g *gamestate.GameState) {
```
Comment (Neelesh99) [inline internal/server/iigointernal/utilities.go:73]: Still the greatest line of code ever !!

**Review by Neelesh99 — APPROVED** [review]
LGTM

File: internal/server/iigointernal/orchestration.go (hunk 1 above)
Comment (Spruudis) [reply internal/server/iigointernal/orchestration.go:30]: oh god seeing as we are simulating whole islands there must be a lot of bodies o_O

**Review by Spruudis — APPROVED** [review]
I am once again getting excited about logging

**Pull request merged**